doc/getting_started.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started with Dinit
+
+In this guide we will go through the steps required to set up a tiny user-mode
+dinit instance.
+
+We assume that dinit has already been installed; i.e. we only cover
+configuration here.
+
+## Starting Dinit
+
+Let's start by running dinit with the bare minimum configuration. First we need
+to choose a place for our service descriptions to live. For this guide we are
+going to use `~/dinit`, but you can use wherever you like.
+
+```
+$ mkdir ~/dinit
+$ cd ~/dinit
+```
+
+Next we need a service description for the `boot` service. This is the service
+that dinit will bring up upon starting.
+
+In a file called `boot` (under the directory we just made) put:
+```
+type = internal
+waits-for.d = boot.d
+```
+
+The first line tells dinit that this is not a service which will run anything
+directly.
+
+The second line specifies a directory which will be used to keep track of which
+services should be enabled at dinit start time. Here we've made it a `boot.d`
+sub-directory under our main dinit directory, but it can be anywhere you like.
+Remember to replace `your-username` with your actual username.
```

Comment:
I think `your-username` was possibly in an ealier draft - doesn't appear in the text?

Reply:
Yes.